utils/g2c_compare.c

```diff
@@ -0,0 +1,91 @@
+/** @file
```

Comment:
I think there's something wrong with this test-- If in the test directory I run `../utils/g2c_compare -v data/gdaswave.t00z.wcoast.0p16.f000.grib2 data/gdaswave.t00z.wcoast.0p16.f000.grib2` but change the name of the first file to one that doesn't exist, then it returns 202. But if I set the second file name to one that doesn't exist, it returns 0. Is that intentional?

Reply:
No that's not expected.

I will add better error handling in the next iteration...